docs/UserGuide.md

```diff
@@ -198,6 +223,11 @@ Format: `add_session n/STUDENT_NAME d/DATE t/TIME k/DURATION s/SUBJECT f/FEE`
 * `DURATION` should be in minutes
 * `FEE` should be the total tuition fee for the total duration
 
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+TutorBuddy takes care of overlapping session for you so that you don't have to worry about it.<br>
+(Overlapping session will return an error)
```

Comment:
Not too sure if the 'Overlapping session will return an error' is necessary. But the 'takes care of overlapping session' also sounds very ambiguous (like we have some algo or something).

Perhaps phrase it in a way like 'TutorBuddy takes care of overlapping session for you by giving a gentle prompt, so you don't have to worry about it'

Reply:
Yeap sounds better! Have updated.